config/discovery-app.container

```diff
@@ -9,7 +9,7 @@ After=discovery-server.service
 
 [Container]
 ContainerName=discovery-app
-Image=quay.io/quipucords/quipucords-ui:latest
+Image=quay.io/quipucords/quipucords-ui:1.7
```

Comment:
mmm only problem here is that there's no UI 1.7. We won't have one for 1.8 either.